recipes/taskflow/all/test_package/conanfile.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+
```

Comment:
```suggestion
```

shebang is not allowed, CCI bot will complain due https://github.com/conan-io/conan-center-index/wiki/Error-Knowledge-Base#kb-h025-meta-lines